README.md

```diff
@@ -1 +1,44 @@
-https://abbybudgetapp.herokuapp.com/
\ No newline at end of file
+# BUDGETLLY
+- A budgetting mobile app that allows the user to add a category and to add expenses that are associatted with the category.
+- This is a Ruby on Rails application that allows the user to:
+
+      - register and log in, so that the data is private to them.
+      - introduce new transactions associated with a category.
+      - see the money spent on each category.
+
+## Getting Started
+-Download the repository as a zip file/ clone the repository using 
+git clone https://github.com/AbbyNyakara/BudgetApp.git
+
+Create a branch and start working!
+
+## Links
+ - [Presentation Video](https://www.loom.com/share/46aa1945875b43e1938cdea5442af9e8)
+ - [Live](https://calm-spire-10060.herokuapp.com/)
+
```

Comment:
- [ ] Let's try to follow as much as possible, the given design, including: Colors, Typography( Font face, size and weight) and Layout( Composition and space between elements)
- Your design
![Screenshot from 2022-11-04 10-16-58](https://user-images.githubusercontent.com/65860678/199926694-806fcdb5-fe75-4d22-87ba-0a11a420a0ad.png)

- Given design
![Screenshot from 2022-11-04 10-25-27](https://user-images.githubusercontent.com/65860678/199926967-cae5951e-136f-4321-84c3-66174dcefbc2.png)

Reply:
hi @uwadonat , that page you have attached is the details page. What you have attached is the transactions page: which looks like this: 
![image](https://user-images.githubusercontent.com/81410040/199966992-ab09389c-f0dc-45e8-9cd3-8ad7dcf2c2b3.png)

Reply:
I also added the delete button to provide a way of deleting a transaction. There is a place to delete a specific expense as per the design, but not for the transactions